ITEM.tsv

```diff
@@ -5553,20 +5553,20 @@ ITEM_20150717_005552	There is not much like this to have as your guard.
 ITEM_20150717_005553	If you take a look at this staff, you can see a great transformation has simplified it. This does not mean the power has been greatly reduced.
 ITEM_20150717_005554	- All Cryomancer Skill Levels increased by +1
 ITEM_20150717_005555	- Earthquake Skill Levels increased by +3
-ITEM_20150717_005556	
-ITEM_20150717_005557	
+ITEM_20150717_005556    The catastrophic effect of this equipment is activated independently from the user's skill.
+ITEM_20150717_005557    From all the people that have used this equipment, we are rooting for you to become the most renowned throughout history.	
 ITEM_20150717_005558	- Basic Attacks have 1% chance to decrease Magic Defense and increase Physical Defense for 30 seconds
 ITEM_20150717_005559	- Basic Attacks have 2% chance to ignite the target for 10 seconds
-ITEM_20150717_005560	
-ITEM_20150717_005561	
-ITEM_20150717_005562	
-ITEM_20150717_005563	
-ITEM_20150717_005564	
-ITEM_20150717_005565	
-ITEM_20150717_005566	
-ITEM_20150717_005567	
-ITEM_20150717_005568	
-ITEM_20150717_005569	
+ITEM_20150717_005560    We have found death without discretion. It is time to apply what we have discovered on the enemy.	
+ITEM_20150717_005561    You can't always draw power from wandering souls. However, if you succeed an item which increases your hidden potential will be created. 	
+ITEM_20150717_005562    The craftsman takes pride in bringing out the optimum curvature of this recurve bow.
+ITEM_20150717_005563    The Star Tower archers do not care for this bow. However, others living in these treacherous times will find this bow useful. 	
+ITEM_20150717_005564    Just because it is called the sniper's bow doesn't mean you have to take single shots.	
+ITEM_20150717_005565    During perilous times, if secret and terror are wielded in the shooter's hands, they will draw out the will to live.	
+ITEM_20150717_005566    The first time a device was attached to the tip of a bow to increase it's draw power was during Lydia Schaffen's era. The actual creator of the device was a master fletcher, who was also Lydia Schaffen's ally.	
+ITEM_20150717_005567    This bow requires a lot more strength from the user than expected. The view of the arrow in flight will leave a sense of satisfaction.	
+ITEM_20150717_005568    Unlike its name, it will be welcomed by any archer. Only rogues dislike it.	
+ITEM_20150717_005569    The burden of two handed bows has been alleviated, it does not decrease the power by much.	
```

Comment:
@"all bows are two handed": I think this is to differentiate between 2h bows (=bows) and 1h bows (=crossbows). There are a couple of other places where this formulation is used, but I think it's safe to replace "two handed bows" by "bows".